### PR TITLE
Workspace and org

### PR DESCRIPTION
**Pull request by captain-Akshay**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — APPROVED** [review]
This is working well in my tests.

It would be ideal to see a screenshot of the empty state (of an expired session or missing org id).

**Comment by leecalcote** [comment]
@captain-Akshay Here is my experience while using the Local Provider:

Screenshot 2024-05-07 at 2 53 41 PM

**Comment by captain-Akshay** [comment]
Duplication is an issue I will try to fix it

**Comment by betelgeuse-code** [comment]
> Duplication is an issue I will try to fix it

This duplication happens because of bad logic in the server itself, whenever Meshery boots it registers a new org, @captain-Akshay this is server specific issue not on UI

File: ui/components/SpacesSwitcher/SpaceSwitcher.js [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+  );
+}
+
+const useStyles = makeStyles((theme) => ({
+  betaBadge: { color: '#EEEEEE', fontWeight: '300', fontSize: '0.8125rem' }, // 13px converted to rem
+  pageTitle: {
+    paddingLeft: theme.spacing(2),
+    fontSize: '1.25rem',
+    [theme.breakpoints.up('sm')]: { fontSize: '1.65rem' },
+  },
```
Comment (betelgeuse-code) [inline ui/components/SpacesSwitcher/SpaceSwitcher.js:208]: We are avoiding to use makeStyles @captain-Akshay

**Review by betelgeuse-code — APPROVED** [review]
LGTM

**Comment by leecalcote** [comment]
> > Duplication is an issue I will try to fix it
> 
> This duplication happens because of bad logic in the server itself, whenever Meshery boots it registers a new org, @captain-Akshay this is server specific issue not on UI

Where has the issue been filed to fix the poor Server behavior? Merging this PR only exacerbates exposure of this poor behavior.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Must use styled components.

This is something that @captain-Akshay already knows (or at least has already been informed of multiple times).

**Comment by leecalcote** [comment]
> Must use styled components.
> 
> This is something that @captain-Akshay already knows (or at least has already been informed of multiple times).

@captain-Akshay please fix today.

**Comment by leecalcote** [comment]
> > Duplication is an issue I will try to fix it
> 
> This duplication happens because of bad logic in the server itself, whenever Meshery boots it registers a new org, @captain-Akshay this is server specific issue not on UI

Open a new issue, please.

**Comment by Yashsharma1911** [comment]
> > > Duplication is an issue I will try to fix it
> > 
> > 
> > This duplication happens because of bad logic in the server itself, whenever Meshery boots it registers a new org, @captain-Akshay this is server specific issue not on UI
> 
> Open a new issue, please.

Issue is alreay being opened here - https://github.com/meshery/meshery/issues/10594

**Comment by Yashsharma1911** [comment]
> > > > Duplication is an issue I will try to fix it
> > > 
> > > 
> > > This duplication happens because of bad logic in the server itself, whenever Meshery boots it registers a new org, @captain-Akshay this is server specific issue not on UI
> > 
> > 
> > Open a new issue, please.
> 
> Issue is alreay being opened here - #10594, I will raise a fix today

I raised a fix Here - https://github.com/meshery/meshery/pull/10899/files

**Comment by leecalcote** [comment]
Is there an open PR in Meshery Extensions?

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
@captain-Akshay what Meshery Docs did you update based on this change?

**Review by leecalcote — CHANGES_REQUESTED** [review]
Build warning:

```
./components/Header.js
457:21  Warning: Missing "key" prop for element in iterator 
```

**Pull request merged**